src/show/pages/public/Home/index.jsx

```diff
@@ -1,19 +1,16 @@
 import { ConstructorCard, DriverCard } from '../../../components';
+import { useEffect, useState } from 'react';
+import axios from 'axios';
```

Comment:
Sorted and grouped incorectly.

`src/show/pages/public/Standings/index.jsx` - you've done well here tho...